.github/CODEOWNERS

```diff
@@ -30,42 +30,35 @@
 /sdk/ @xirzec @Azure/azure-sdk-js-dev
 
 # PRLabel: %Azure.Core
-# ServiceLabel: %Azure.Core
-# AzureSdkOwners: @Azure/azure-sdk-for-js-core
 /sdk/core/ @Azure/azure-sdk-for-js-core
 
 # PRLabel: %Azure.Core
-# ServiceLabel: %Azure.Core
-# AzureSdkOwners:  @jeremymeng @deyaaeldeen @HarshaNalluru
 /sdk/core/core-amqp/ @jeremymeng @deyaaeldeen @HarshaNalluru
 
 # PRLabel: %Azure.Core
-# ServiceLabel: %Azure.Core
-# AzureSdkOwners: @joheredi @jeremymeng @deyaaeldeen
 /sdk/core/core-client-rest/ @joheredi @jeremymeng @deyaaeldeen @timovv
 
 # PRLabel: %Azure.Core
-# ServiceLabel: %Azure.Core
-# AzureSdkOwners: @deyaaeldeen @joheredi
 /sdk/core/core-lro/ @deyaaeldeen @joheredi
 
 # PRLabel: %Azure.Core
-# ServiceLabel: %Azure.Core
-# AzureSdkOwners: @maorleger @jeremymeng @mpodwysocki
 /sdk/core/core-tracing/ @maorleger @jeremymeng @mpodwysocki
 
-# Service teams
-# ServiceLabel: %App Configuration
+# ServiceLabel: %Azure.Core
+# AzureSdkOwners: @Azure/azure-sdk-for-js-core
+
 # PRLabel: %App Configuration
-# AzureSdkOwners:  @HarshaNalluru @minhanh-phan
 /sdk/appconfiguration/ @HarshaNalluru @minhanh-phan
 
-# ServiceOwners:  @anilba06 @gkostal
-# ServiceLabel: %Attestation
+# ServiceLabel: %App Configuration
+# AzureSdkOwners:  @HarshaNalluru @minhanh-phan
```

Comment:
@minhanh-phan  @HarshaNalluru do we have a contact for app config # ServiceOwners field?

Reply:
Maybe @jimmyca15.

Any other contacts @jsquire @ronniegeraghty ?

Reply:
I'd copy whomever is hanging out in the .NET CODEOWNERS.   I don't believe their PM team is active on GH, so Jimmy may be the best idea.

Reply:
@zhenlan and myself sounds reasonable.

Reply:
Thanks all! updated.